azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -732,6 +733,19 @@ public void shutdownNow() {
     logger.warn("Shutting down FlowRunnerManager now...");
     this.executorService.shutdownNow();
     this.triggerManager.shutdown();
+    deleteExecutionDirectory();
```

Comment:
@chengren311 so you're saying I should call this in the AzkabanExecutorServer's shutdownInternal/shutdownNow methods instead of in the FlowRunnerManager?

I figure it's better to do it here because the sequencing (all running jobs end, then delete execution directory) is more clear. What do you think?